third_party/2and3/dateutil/rrule.pyi

```diff
@@ -0,0 +1,103 @@
+from ._common import weekday as weekdaybase
+from typing import Any, List, Optional, Union
+import datetime
+
+YEARLY: int
+MONTHLY: int
+WEEKLY: int
+DAILY: int
+HOURLY: int
+MINUTELY: int
+SECONDLY: int
+
+class weekday(weekdaybase):
+    def __init__(self, wkday, n: Optional[Any] = ...) -> None: ...
+
+MO: weekday
+TU: weekday
+WE: weekday
+TH: weekday
+FR: weekday
+SA: weekday
+SU: weekday
+
+class rrulebase:
+    def __init__(self, cache: bool = ...) -> None: ...
+    def __iter__(self): ...
+    def __getitem__(self, item): ...
+    def __contains__(self, item): ...
+    def count(self): ...
+    def before(self, dt, inc: bool = ...): ...
+    def after(self, dt, inc: bool = ...): ...
+    def xafter(self, dt, count: Optional[Any] = ..., inc: bool = ...): ...
+    def between(self, after, before, inc: bool = ..., count: int = ...): ...
+
+class rrule(rrulebase):
+    def __init__(self,
+                 freq,
+                 dtstart: Optional[datetime.datetime] = ...,
+                 interval: int = ...,
+                 wkst: Optional[Union[weekday, int]] = ...,
+                 count: Optional[int] = ...,
+                 until: Optional[Union[datetime.datetime, int]] = ...,
+                 bysetpos: Optional[Union[int, List[int]]] = ...,
```

Comment:
The docs for all of these say they accept Sequences, but I think it actually accepts `Iterable[int]`. Can you change them all to take `Iterable[int]`?